rust/flatbuffers/src/builder.rs

```diff
@@ -88,6 +88,33 @@ impl<'fbb> FlatBufferBuilder<'fbb> {
         }
     }
 
+    /// The current size of the builder's buffer in bytes, counting from the end.
+    ///
```

Comment:
Please add a note about who the intended user would be. (i.e. experts with their own needs.)